internals/reminders_test.py

```diff
@@ -174,7 +174,7 @@ def test_build_email_tasks_feature_accuracy__enterprise(self):
     self.assertEqual('feature_owner@example.com', task['to'])
     self.assertEqual('[Action requested] Update feature one', task['subject'])
     self.assertEqual(None, task['reply_to'])
-    TESTDATA.make_golden(task['html'], 'test_build_email_tasks_feature_accuracy_enterprise.html')
+    # TESTDATA.make_golden(task['html'], 'test_build_email_tasks_feature_accuracy_enterprise.html')
```

Comment:
Thanks for catching this mistake.  That effectively disabled a test.

Reply:
I believe I was the last one to edit this, so naturally I'm fixing my own mistake. 😅